models/event-videos-post.json

```diff
@@ -11,6 +11,10 @@
       "description": "The unique url of the video. Currently only Youtube and Vimeo sources are allowed.",
       "pattern": "^http(s?):\\/\\/(www\\.)?((youtube\\.com\\/watch\\?v=([^\\/#&?]*))|(vimeo\\.com\\/([^\\/#&?]*)))"
     },
+    "language": {
+      "type": "string",
+      "example": "nl"
```

Comment:
We can also specify a minimum length here I think of 2, maybe also even a maximum length of 2.

Reply:
In other places I've documented it as an enum with `nl`, `fr`, `de`, `en` as possible values to keep it simple and easy to understand. Of course this would effectively block any other languages in the validation on the backend. An alternative could be a regex pattern that only matches with strings of exactly two characters that are both in a-z.
For example: `^[a-zA-Z][a-zA-Z]$` (to keep it case-insensitive, because we can easily do a `strtolower()` on it to normalize it before we store it)

Reply:
I am not 100% sure if events & places can now only be in Dutch, French, German or English to be honest. Via the UI maybe, but on the API it's possible that any language code is allowed. We'll have to investigate that. But for now an enum is fine yes. 👍